cmd/admin-handlers.go

```diff
@@ -329,6 +329,15 @@ type ServerMemUsageInfo struct {
 	HistoricUsage []mem.Usage `json:"historicUsage"`
 }
 
+// ServerNetStatsInfo holds information about network throughput
```

Comment:
Need to fix comment here and a few other places too

Reply:
done